src/scripts/py2_code_hunter/py2_code_hunter.py

```diff
@@ -9,8 +9,8 @@
 from lib2to3.refactor import RefactoringTool, get_fixers_from_package
```

Comment:
Is this script and its test even worth keeping any more?

Reply:
I hope not!

Reply:
It was really used to notify users who had py2 code in their narratives way back before some major Python / IPython version switchover to make those fail. I think it hit single-digits of non-KBase-staff.